lib/sdk.ts

```diff
@@ -20,8 +21,15 @@ import { sha256 } from "js-sha256";
 class OptableSDK {
   public sandbox: OptableConfig; // legacy
 
-  constructor(public dcn: OptableConfig) {
+  constructor(public dcn: OptableConfig, writePassport: boolean = true) {
     this.sandbox = dcn; // legacy
+    writePassport && Init(dcn);
```

Comment:
Init() returns a promise that nobody waits on right now, there is no guarantee that the passport is written after constructing an OptableSDK. Is that intentional?

Since it's not possible to return a promise from a constructor I'd suggest to leave the constructor as is (backward compat), instead introduce a new static function acting as a contructor that will internally call the constructor and `await` for `init()` before returning the sdk instance.

Something like:

```
public static async Init(dcn: OptableConfig): Promise<OptableSDK> {
  const sdk = new OptableSDK(dcn);
  await sdk.init();
  return sdk;
}
```

usage

```
OptableSDK.Init(dcn).then((sdk) => sdk.identify(...))
```
but maybe I'm missing info